symplyphysics/laws/astronomy/penetrating_power_of_telescope.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve, log, S,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to
+)
+
+# Description
+## The penetrating force is the magnitude of the faintest stars visible with a telescope when observed at the zenith.
+
+## Law is: m = 2.5 + 5 * lg(D), where
+## m - penetrating power of telescope,
+## D - diameter of the lens in millimeters.
+
+penetrating_power_telescope= Symbol("penetrating_power_telescope", dimensionless)
+
+lens_diameter = Symbol("lens_diameter", units.length)
+
+law = Eq(penetrating_power_telescope, 2.5 + 5 * log(lens_diameter / Quantity(1 * units.millimeter),10))
```

Comment:
Extract this quantity to separate variable: `one_millimeter = Quantity(1 * units.millimeter)`